### PR TITLE
Add encryption parameters to the APIs

### LINKED ISSUE #36: JSON-RCP API has supports encrypted data
CreateAccount, CreateMoneyOrder, RedeemMoneyOrder methods have now extra fields: `encryptedX` and `randomizationFactor`

### PR DESCRIPTION
**Pull request by bgravenorst**
## Pull Request Description
Add encryption parameters to the APIs

## Fixed Issue(s)
Fixes #36 

## Preview docs
https://pegasys-sumo--45.com.readthedocs.build/en/45/Reference/JSON-RPC/

### CODE REVIEW

File: docs/Reference/JSON-RPC.md [hunk 1]
```diff
@@ -18,11 +18,17 @@ Creates an account on the rollup.
 
 * `publicKey`: *string* - 64-byte account public key
 
-* `encryptionKey`: *string* - 64-byte encryption key associated with the supplied public key
+* `jointEncryptionKey`: *string* - 128-bytes encryption key based on the user's public key and
```
Comment (alexandratran) [inline docs/Reference/JSON-RPC.md:21]:
```suggestion
* `jointEncryptionKey`: *string* - 128-byte encryption key based on the user's public key and
```

File: docs/Reference/JSON-RPC.md [hunk 2]
```diff
@@ -151,7 +162,8 @@ Returns the latest finalized account state on the blockchain.
 
 * `publicKey`: *string* - account public key
 
-* `encryptionKey`: *string* - 64-byte encryption key associated with the public key
+* `jointEncryptionKey`: *string* - 128-bytes encryption key based on the user's public key and
```
Comment (alexandratran) [inline docs/Reference/JSON-RPC.md:165]:
```suggestion
* `jointEncryptionKey`: *string* - 128-byte encryption key based on the user's public key and
```

**Review by alexandratran — APPROVED** [review]
The API page has inconsistent usage of full stops at the end of the parameter bullet points, otherwise LGTM.

**Comment by bgravenorst** [comment]
Due to issues with the PR after the renaming the repo, all changes for this issue is implemented in https://github.com/ConsenSys/doc.rollups/pull/48

**Pull request closed without merging**